backend-project/src/Ex0502_02.java

```diff
@@ -0,0 +1,23 @@
+import java.util.*;
+
+public class Ex0502_02 {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		Scanner sc = new Scanner(System.in);
+
+		System.out.print("�� �� �Է��ϼ��� : ");
+		int i = sc.nextInt();
+
+		if (i <= 0) {
```

Comment:
논리 연산자 OR을 사용하시면 if 문 하나로 예외처리 하실 수 있습니다! 
```suggestion
		if (i <= 0 || i >= 1000) {
```
요런 식으로요~

Reply:
알려주신 부분 예외처리 코드에 적용했습니다!